gradle.properties

```diff
@@ -16,3 +16,6 @@ POM_DEVELOPER_ID=jakewharton
 POM_DEVELOPER_NAME=Jake Wharton
 
 org.gradle.jvmargs=-Xmx1536M
+
+# needed so that :timber:prepareLintJarForPublish can succeed
+kotlin.stdlib.default.dependency=false
```

Comment:
is there a bug report for this somewhere so we know when to remove this workaround?

Reply:
I think it had something to do with [this](https://stackoverflow.com/a/66966374/691639). I don't remember the exact issue, but it was causing some sort of conflict.

Reply:
As far as I know Lint is enforcing a global kotlin version which at least for AGP 4.2 was 1.3.72 and you weren't allowed to add any other version. This probably implied adding a version other than the one Lint is enforcing

Reply:
This is the error:

```
Execution failed for task ':timber:prepareLintJarForPublish'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.PublishLintJarWorkerRunnable
   > Found more than one jar in the 'lintPublish' configuration. Only one file is supported. If using a separate Gradle project, make sure compilation dependencies are using compileOnly

```

Reply:
got it.  can we move this to timber-lint like so?

`timber-lint/gradle.properties`
```
# needed so that :timber:prepareLintJarForPublish can succeed
# Remove when the bug described in https://issuetracker.google.com/issues/161727305 is fixed
kotlin.stdlib.default.dependency=false
```

Reply:
Done